greedy_method/minimum_number_of_refills.py

```diff
@@ -0,0 +1,25 @@
+arr = []
+B = int(input())    #maximum distance traversed in 1 full tank
+L = int(input())    #destination
+n=int(input())
+arr = list(map(int, input().split()))
+
+arr.insert(0,int(0))
+arr.append(B)
+
+def MinRefills(x,n,L):
```

Comment:
Single letter variable names are old school as discussed in CONTRIBUTING.md
* No python type hints
* No doctests